### PR TITLE
search: codenotify @rvantonder

### PR DESCRIPTION
**Pull request by rvantonder**
I'm missing out on Go search PRs I care about, so doing a couple of CODENOTIFYs :-)

### CODE REVIEW

File: cmd/frontend/graphqlbackend/CODENOTIFY [hunk 1]
```diff
@@ -6,6 +6,8 @@ site_monitoring.go @bobheadxi
 # Search
 *search*.go @keegancsmith
 *zoekt*.go @keegancsmith
+*search*.go @rvantonder
+*zoekt*.go @rvantonder
```
Comment (keegancsmith) [inline cmd/frontend/graphqlbackend/CODENOTIFY:10]: you can add your name next to mine above :)

**Review by keegancsmith — APPROVED** [review]

**Comment by keegancsmith** [comment]
Side-note: How easy would it be to extend our query language to search codeowners now? I can imagine a simple backend which contributes `FileMatch` with just paths for codeowners. You could then and against that and just search files you own :O

**Comment by rvantonder** [comment]
That's a _really_ nice idea. I'm collecting search ideas like that to drive how we would extend things and support certain "builtins", and that'll definitely be on the list! More soon. For this use case we'd  essentially need a primitive to substitute the content of a match for a search pattern.

**Comment by lguychard** [comment]
That would be 🤯 🤩. Tracking in https://github.com/sourcegraph/sourcegraph/issues/17311

**Pull request merged**